packages/ai-rules/scripts/buildProfile.ts

```diff
@@ -3,9 +3,8 @@ import { copyFile, cp, mkdir, mkdtemp, rm } from "node:fs/promises";
 import { tmpdir } from "node:os";
 import { join } from "node:path";
 
-import { toErrorMessage } from "@clipboard-health/util-ts";
-
 import { PATHS, type ProfileName } from "./constants";
+import { toErrorMessage } from "./toErrorMessage";
```

Comment:
Why did we stop using `@clipboard-health/util-ts`?

Is it to ensure that ai-rules does not have any deps when installed?